dask/array/core.py

```diff
@@ -1776,6 +1776,22 @@ def cumprod(self, axis, dtype=None, out=None):
         from .reductions import cumprod
         return cumprod(self, axis, dtype, out=out)
 
+    @derived_from(np.ndarray)
+    def item(self, *args):
+        if len(args) == 0:
+            if self.size != 1:
+                raise ValueError(
+                    "Only size 1 arrays can be converted to scalars."
+                )
+            args = self.ndim * (0,)
+        elif len(args) == 1:
+            if isinstance(args, tuple) and isinstance(args[0], tuple):
+                args = args[0]
+            else:
+                args = np.unravel_index(args[0], self.shape)
+
+        return self[args].astype(object)
```

Comment:
This is a size 0 scalar dask array?

Reply:
From playing around with it locally, seems to be.

Though now that you ask this am thinking we need a check to ensure `args` has the same length as `self.ndim`.

Are there any other cases we should be checking?

Reply:
The slicing gets the scalar. The `object` conversion seems to be needed due to a type mismatch that occurs otherwise between the NumPy and Dask implementations (e.g. `float` vs. `float64`).